common/Views/CloseButton.xaml

```diff
@@ -0,0 +1,18 @@
+<Button
+    x:Class="DevHome.Common.Views.CloseButton"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Style="{StaticResource DefaultButtonStyle}"
+    HorizontalAlignment="Right"
+    VerticalAlignment="Top"
+    BorderThickness="0"
+    Background="Transparent"
+    Padding="10">

```

Comment:
We will need to add a localized AutomationProperties.Name to this control later, as the narrator will read it as "Button" and nothing else.